scripts/dev/run_python.sh

```diff
@@ -16,8 +16,9 @@ fi
 
 export PYTHONPATH="${PROJECT_DIR}"
 
-current_python_version=$(python --version 2>&1 | awk '{split($2, a, "."); print a[1] "." a[2]}')
-if [[ "${current_python_version}" != "${PYTHON_VERSION}" ]]; then
+current_python_version=$(python --version 2>&1 | awk '{print $2}')
+required_version="${PYTHON_VERSION:-3.13.7}"
```

Comment:
if `PYTHON_VERSION` is not set, we should exit the script, not using the default value.

Reply:
yeah will be handled the same everywhere